tests/js/list-tests.js

```diff
@@ -1840,58 +1840,42 @@ module.exports = {
 
   testClassObjectCreation: function () {
     class TodoItem extends Realm.Object {
-      constructor(description) {
-        super();
-        this.id = new ObjectId();
```

Comment:
Why remove `id`?  What is the equivalent way to do this now?

Reply:
Since `id` was a primary key I got errors (duplicated primary key) when creating a new object from a managed object here: https://github.com/realm/realm-js/blob/kh/class-models-object-constructor/tests/js/list-tests.js#L1878

I can add it back in but remove the primary key?

Reply:
Ok thats fine, then there's really no need for it.